index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#">Joaquin Calle Aranzolo</a>
+            <i class="fa fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <h1>Hey there. I'm Joaquin.</h1>
+        <h3>I'm a Software Developer</h3>
+        <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you needed coded, don't hesitate to contact me.</p>
+        <ul class="icon-bar extra-bar">
+            <li>
+                <a href="#"><img src="./img/Twitter.png" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="./img/Linkedin.png" alt="Linkedin"></a>
+            </li>
+            <li>
+                <a href="#"><img src="./img/Github.png" alt="Github"></a>
+            </li>
+            <li>
+                <a href="#"><img src="./img/Angellist.png" alt="Angellist"></a>
+            </li>
```

Comment:
- [ ] As requested by the previous reviewer please update that, your `href="#"` should be 

```html
<a href="https//github/yourname.com" >Github</a>
```

and for the others please update them with your credentials